modules/gdscript/editor_templates/EditorScript/basic_editor_script.gd

```diff
@@ -2,6 +2,7 @@
 @tool
 extends EditorScript
 
+
+# Called when the script is executed (using File -> Run in Script Editor).
 func _run() -> void:
-    # Called when the script is executed (using File -> Run in Script Editor).
     pass
```

Comment:
You seem to be converting all spaces to tabs but forgot this line. Shouldn't the templates use `_TS_` so the indentation is converted to spaces/tabs based on the user configuration? I don't know how that works.

Reply:
Good catch, thanks. From my testing, it seems that it converts the tabbing without the need for `_TS_`.